### ISSUE TITLE
Find().actualView gets type mismatch error

### ISSUE THREAD

**Issue by pobengtsson** [nalexn/ViewInspector#104]
Hi Alexey,

Thanks for all your work creating ViewInspector!

I have an issue using find together with actual view. Seemingly, when two views in a hierarchy have names that both starts the same, the following code results in a error.

given code like this:
```swift
struct ContentView: View {
    var body: some View {
         FooList()
    }
}

struct FooList: View {
   var body: some View {
      ForEach (0..<5) { item in
         Foo()
      }
   }
}

struct Foo: View {
   var body: some View {
      Text("bar")
   }
}

extension ContentView: Inspectable {}
extension Foo: Inspectable {}
extension FooList: Inspectable {}
```

The following line from the test code:
```swift
        _ = try view.inspect().find(Foo.self).actualView()
```
results in:
`failed: caught error: "Type mismatch: FooList is not Foo"`

I made a minimal project that should reproduce the issue, at least on my machine ;) 
[findReproduceBug.zip](https://github.com/nalexn/ViewInspector/files/6404404/findReproduceBug.zip)

What do you think about the issue?

**Comment by nalexn** [comment]
Hey @pobengtsson that's a bug, thanks for finding and reporting! Will be fixed in the coming update

**Comment by pobengtsson** [comment]
Great! Thanks @nalexn!

**Comment by nalexn** [comment]
Fixed in v0.7.6. Thanks for the report!
